<launch>
  <arg name="robot" default="invite_motoman_moveit_config"/>

  <arg name="group"        default="arm_left"/>
  <arg name="end_effector" default="left_gripper"/>


  <node pkg="moveit_simple_grasps" type="moveit_simple_grasps_server" name="moveit_simple_grasps_server">
    <param name="group"        value="$(arg group)"/> 
    <param name="end_effector" value="$(arg end_effector)"/>

    <rosparam command="load" file="$(find invite_motoman_moveit_config)/config/gripper_grasp.yaml"/> 

  </node>


</launch>
